cardano-cli/src/Cardano/CLI/Legacy/Options.hs

```diff
@@ -39,6 +39,7 @@ import           Cardano.CLI.Legacy.Commands.TextView
 import           Cardano.CLI.Legacy.Commands.Transaction
 import           Cardano.CLI.Parser
 import           Cardano.CLI.Types.Common
+import           Cardano.CLI.EraBased.Options.Governance.Poll
```

Comment:
Restore the original legacy parsers instead of using the era-based one.

Reply:
Done :+1: